.eslintrc.yml

```diff
@@ -212,7 +212,7 @@ rules:
   no-label-var:
     - error
   no-shadow:
-    - error
+    - off
```

Comment:
Just out of sheer curiosity - what does this change mean? Would it be better to add exclusions in the one or two places that need it?